#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("angular_resolution", double_t, 0, "Angular resolution", 0.5, 0.001, 1.0)
gen.add("noise_level", double_t, 0, "Noise level", 0.1, 0.1, 1.0)
gen.add("min_range", double_t, 0, "minimum range", 0.0, 0.0, 10.0)
gen.add("border_size", int_t, 0, "border size", 0, 0, 10)
gen.add("max_angle_height", double_t, 0, "maximum angle height", 2 * pi, 0, 2 * pi)
gen.add("max_angle_width", double_t, 0, "maximum angle width", 2 * pi, 0, 2 * pi)
exit (gen.generate (PACKAGE, "jsk_pcl_ros", "BorderEstimator"))
